libcudacxx/include/cuda/std/__type_traits/is_extended_floating_point.h

```diff
@@ -53,7 +53,7 @@ _CCCL_INLINE_VAR constexpr bool __is_extended_floating_point_v
 #  endif // !_CCCL_NO_INLINE_VARIABLES
 #endif // !_CCCL_NO_VARIABLE_TEMPLATES
 
-#if defined(_LIBCUDACXX_HAS_NVFP16)
+#if defined(_CCCL_HAS_FP16)
```

Comment:
This will not work, because above we guard with `_LIBCUDACXX_HAS_NVFP16`

Please also change the include guard

Reply:
> This will not work, because above we guard with `_LIBCUDACXX_HAS_NVFP16`

Fixes

> Please also change the include guard

Can you elaborate? The include guard `_LIBCUDACXX___TYPE_TRAITS_IS_EXTENDED_FLOATING_POINT_H` looks good to me.

Reply:
Oh I meant please also change theguard for the `<cuda_fp16.h>` include